File: Python/1015.py
import math

p1 = input().split()
x1 = float(p1[0])
y1 = float(p1[1])

p2 = input().split()
x2 = float(p2[0])
y2 = float(p2[1])

distancia = math.sqrt((x1-x2)**2+(y1-y2)**2)

print(f'{distancia:.4f}')

File: Ruby/1_Iniciante/1015.rb
p1 = gets.split()
x1 = p1[0].to_f
y1 = p1[1].to_f

p2 = gets.split()
x2 = p2[0].to_f
y2 = p2[1].to_f

distancia = Math.sqrt((x1-x2)**2+(y1-y2)**2)

puts "%.4f" % distancia
